src/components/hosts/utils.ts

```diff
@@ -30,6 +30,9 @@ export const canEditRole = (clusterStatus: Cluster['status'], status: Host['stat
     'pending-for-input',
   ].includes(status);
 
+export const canDownloadKubeconfig = (clusterStatus: Cluster['status']) =>
+  ['ready', 'installing', 'finalizing', 'installed'].includes(clusterStatus);
```

Comment:
I don't think that we have kubeconfig when cluster is ready.

Reply:
thanks!